ci-operator/step-registry/openshift/e2e/aws/ovn/upgrade/loki/openshift-e2e-aws-ovn-upgrade-loki-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: openshift-e2e-aws-ovn-upgrade-loki
```

Comment:
Should be `openshift-upgrade-e2e-aws-ovn-loki` to match other upgrade workflows

Reply:
that would mean moving this file to ci-operator/step-registry/openshift/upgrade/e2e/aws/ovn/loki

not a problem for me, but ci-operator/step-registry/openshift/ovn/ seemed like a more relevant root folder.

let me know which is best and I'll change it if needed.

Reply:
I think this should be `openshift-upgrade-...` as all of the other upgrade workflows use that. IMO we should make it consistent. `ci-operator/step-registry/openshift/upgrade/e2e/aws/ovn/loki` is fine as a root folder.